src/webview_provider/insightsPanel.ts

```diff
@@ -193,6 +193,9 @@ export class InsightsPanel extends AltimateWebviewProvider {
           },
         });
       }
+      await this.dbtProjectContainer
+        .findDBTProject(Uri.parse(params.projectRoot))
```

Comment:
Does projectRoot contains schema? Would it work with `Uri.parse(params.projectRoot, true)`?

Reply:
its absolute path. and not working with `Uri.parse(params.projectRoot, true)`